docs/topics/braintree.rst

```diff
@@ -227,6 +227,17 @@ solitude.
 
     :status 201: payment method created.
 
+Cancel a subscription. This will cancel the subscription in Braintree and is
+not reversible.
+
+.. http:post:: /braintree/subscription/cancel/
+
+    :<json string paymethod: the resource_uri of the subscription in solitude.
```

Comment:
I'm still thinking that a PUT on the subscription resource would make sense here. There are multiple data objects but that's just an implementation detail. To the caller, there is only one subscription by one ID.